metadata-models/src/main/pegasus/com/linkedin/access/token/DataHubAccessTokenInfo.pdl

```diff
@@ -45,13 +45,14 @@ record DataHubAccessTokenInfo {
   createdAt: Time
 
   /**
-   * When the token was expires.
+   * When the token expires.
    */
+   // if null, it means that the token does not have an expiry.
```

Comment:
Why are we adding comments down here? 

Just add it as a newline in the comment above.

Reply:
Do we really need this detail as a model comment?
This is added separately below so that it is not exposed with model for now, but for developers.
I feel we can bump it up into model comment later if required.